seam/modules/action_attempts.py

```diff
@@ -0,0 +1,67 @@
+from typing import Dict, Optional, Union
+import time
+
+from ..client import SeamHttpClient
+from ..exceptions import SeamActionAttemptFailedError, SeamActionAttemptTimeoutError
+from ..routes.models import ActionAttempt
+
+TIMEOUT = 5.0
+POLLING_INTERVAL = 0.5
+
+
+def get_action_attempt(client: SeamHttpClient, action_attempt_id: str) -> ActionAttempt:
```

Comment:
Instead of accepting `client` param, I tried to use the `ActionAttempts` client instance. However, when I import `from ..routes.action_attempts import ActionAttempts` here, it causes a circular import error again.
![image](https://github.com/seamapi/python-next/assets/84702959/5ceb1ff9-ecda-477b-84e8-090147774cd2)